htslib/hts.h

```diff
@@ -508,6 +508,14 @@ int hts_close(htsFile *fp);
 HTSLIB_EXPORT
 const htsFormat *hts_get_format(htsFile *fp);
 
+/*!
+  @abstract  Returns the file's name
+  @param fp  The file handle
+  @return    Read-only pointer to the file's fn field.
+*/
+HTSLIB_EXPORT
+const char *hts_get_fn(htsFile *fp);
```

Comment:
Are there other parts of the API that have `…fn` in API function names? I'd rather spell it out as `…filename` (and it needs to be clear what if any relationship this has to `hts_set_fai_filename`).